kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/NMDbusConnector.java

```diff
@@ -259,56 +306,66 @@ private synchronized void manageConfiguredInterfaces(List<String> configuredInte
 
     private synchronized void manageConfiguredInterface(String iface, NetworkProperties properties)
             throws DBusException {
-        Device device = getDeviceByIpIface(iface);
-        NMDeviceType deviceType = getDeviceType(device);
-
-        KuraIpStatus ip4Status = KuraIpStatus
-                .fromString(properties.get(String.class, "net.interface.%s.config.ip4.status", iface));
-        // Temporary solution while we wait to add complete IPv6 support
-        KuraIpStatus ip6Status = ip4Status == KuraIpStatus.UNMANAGED ? KuraIpStatus.UNMANAGED : KuraIpStatus.DISABLED;
-        KuraInterfaceStatus interfaceStatus = KuraInterfaceStatus.fromKuraIpStatus(ip4Status, ip6Status);
-
-        if (!CONFIGURATION_SUPPORTED_DEVICE_TYPES.contains(deviceType)
-                || !CONFIGURATION_SUPPORTED_STATUSES.contains(ip4Status)
-                || !CONFIGURATION_SUPPORTED_STATUSES.contains(ip6Status)) {
-            logger.warn("Device \"{}\" of type \"{}\" with status \"{}\"/\"{}\" currently not supported", iface,
-                    deviceType, ip4Status, ip6Status);
-            return;
-        }
+        Optional<Device> device = getDeviceByInterfaceName(iface);
+        if (device.isPresent()) {
```

Comment:
Again, style preference for early return so that you can also add a log entry

```suggestion
        if (!device.isPresent()) {
            log.info("Device \"{}\" is missing. Configuration skipped", iface);
        } 
```